github/actions_variables.go

```diff
@@ -0,0 +1,293 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// Variable represents a repository action variable.
+type Variable struct {
+	Name       string    `json:"name"`
+	Value      string    `json:"value"`
+	CreatedAt  Timestamp `json:"created_at"`
+	UpdatedAt  Timestamp `json:"updated_at"`
+	Visibility string    `json:"visibility,omitempty"`
+	// Used by ListOrgVariables and GetOrgVariables
+	SelectedRepositoriesURL string `json:"selected_repositories_url,omitempty"`
+	// Used by UpdateOrgVariable and CreateOrgVariable
+	SelectedRepositoryIDs SelectedRepoIDs `json:"selected_repository_ids,omitempty"`
```

Comment:
```suggestion
	CreatedAt  *Timestamp `json:"created_at,omitempty"`
	UpdatedAt  *Timestamp `json:"updated_at,omitempty"`
	Visibility *string    `json:"visibility,omitempty"`
	// Used by ListOrgVariables and GetOrgVariables
	SelectedRepositoriesURL *string `json:"selected_repositories_url,omitempty"`
	// Used by UpdateOrgVariable and CreateOrgVariable
	SelectedRepositoryIDs *SelectedRepoIDs `json:"selected_repository_ids,omitempty"`
```

Reply:
As said above, i strictly followed `actions_secrets.go` here! Do you want me to change this anyway?

Reply:
Yes, I believe these changes would be good and/or needed here.

It may be that I allowed errors into actions_secrets.go during my code review, but I hope that is not the case.
I looked at the docs for actions variables, and I believe these changes are necessary, so please let's apply these changes to this PR.

Reply:
And please remember to run "go generate ./..." and push the results once you finish making the changes.